src/editor/objects/SimpleWater.js

```diff
@@ -0,0 +1,250 @@
+import { Mesh, PlaneBufferGeometry, MeshStandardMaterial, MeshPhongMaterial, Vector2, RepeatWrapping } from "three";
```

Comment:
If this file is intended to remain sync 1:1 with the one in hubs (which it looks like it is) we should add a comment here making note of that. It could also be moved to lib-hubs but that's obviously more work.

Reply:
Yeah I want to move `SimpleWater` and `Sky` to `lib-hubs` I'll add the comment for now.